src/libraries/sendtohelixhelp.proj

```diff
@@ -163,14 +161,19 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(HelixCommand)' == ''">
+    <!-- 
+      We are hosting the payloads for the WASM/browser on kestrel in the xharness process.
+      We also run some network tests to this server and so, we are running it on both HTTP and HTTPS.
+      For the HTTPS endpoint we need development SSL certificate.
+    -->
     <!--
       For Windows we need to use "call", since the command is going to be called from a batch script created by Helix.
       We "exit /b" at the end of RunTests.cmd. Helix runs some other commands after ours within the batch script,
       so if we don't use "call", then we cause the parent script to exit, and anything after will not be executed.
     -->
-    <HelixCommand Condition="'$(TargetsWindows)' == 'true' or '$(BrowserHost)' == 'windows'">call RunTests.cmd</HelixCommand>
+    <HelixCommand Condition="'$(TargetsWindows)' == 'true' or '$(BrowserHost)' == 'windows'">dotnet dev-certs https &amp;&amp; call RunTests.cmd</HelixCommand>
```

Comment:
Any reason for not having `dotnet dev-certs https` as a `HelixPreCommand`?

Reply:
See here https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-53277-merge-5628afbfb7314eaeae/System.Net.Http.Json.Functional.Tests/console.c58f5af9.log?sv=2019-07-07&se=2021-06-15T11%3A59%3A29Z&sr=c&sp=rl&sig=KW9SsPKDADoV8OokbFyjERkqajhQNEhFPrBxLk2gfw4%3D

It is inserted before PATH is set to dotnet SDK

Reply:
hm then I would set `<HelixCommand>dotnet dev-certs https &amp;&amp;</HelixCommand`>, and the rest can append to it.